src/main/riddl/Tenant/tenantProjections.riddl

```diff
@@ -1,31 +1,29 @@
+query FindTenantByName is {name: String} briefly "Get a list of tenants that a member belongs to."
```

Comment:
The briefly comment needs to be updated. Maybe, "Get tenant info by searching for it by name."?